src/site/_includes/components/SignPosts.js

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+const {html} = require("common-tags");
+const stripLanguage = require("../../_filters/strip-language");
+
+module.exports = ({postToCollections, url}) => {
+  // When @devnook lands her localization work we might need to update this file.
+  const strippedUrl = stripLanguage(url).replace(/\//g, "");
```

Comment:
Is this stripping all slashes from a page?
What about e.g. https://web.dev/handbook/content-checklist/ ?

Reply:
It's only used so that for the keys of an object from `posts-to-collections.js` which doesn't include handbook urls. Maybe this is something to be addressed later, if at all (I'm not sure)?